docs/workflows/build-pages-with-storybook.md

```diff
@@ -117,13 +117,75 @@ module.exports = {
   // your Storybook configuration
 
   webpackFinal: config => {
-    config.resolve.alias.fetch = '../__mocks__/fetch.js'
+    config.resolve.alias['isomorphic-fetch'] = require.resolve('../__mocks__/isomorphic-fetch.js');
     return config;
    }
 };
 ```
 
-You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story.
+You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story. An example of how that might look:
+
+```js
+// __mocks__/isomorphic-fetch.js
+let nextJson;
+export default async function fetch() {
+  if (nextJson) {
+    return {
+      json: () => nextJson,
+    };
+  }
+  nextJson = null;
+}
+
+export function decorator(story, { parameters }) {
+  if (parameters && parameters.fetch) {
+    nextJson = parameters.fetch.json;
+  }
+  return story();
+}
+```
```

Comment:
@tmeasday makes a bit more sense to move this up before the code snippet in the `.storybook/main`. And we change this:

"You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story. An example of how that might look:"

To:

"Let's start by creating our own mock, which we'll use later with a [decorator](../writing-stories/decorators#global-decorators]. Create a new file called `isomorphic-fetch.js` inside a directory called `__mocks__` (we'll leave the location to you, don't forget to adjust the imports to your needs) and add the following code inside:"

Something like this. WDYT?